pallets/popi/src/lib.rs

```diff
@@ -16,6 +16,8 @@ mod benchmarking;
 pub mod weights;
 pub use weights::*;
 
+use pallet_timestamp::{self as timestamp};
```

Comment:
I believe this is causing the Github check to fail. It's not being used, so you should remove it (or comment it out) until you're ready to upload the `get_time` around line 172.
```suggestion

```

Reply:
good catch!